api/HTMLCollection.json

```diff
@@ -76,10 +76,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "≤4"
```

Comment:
This ideally should have failed the lint. The parent feature was version "4", so "≤4" doesn't make sense, it couldn't have been a release before 4.

I see you added some custom tests in https://github.com/foolip/mdn-bcd-collector/pull/872, will that allow us to check for support in Safari 3 and see if we can resolve this to use either see  "4", or "≤4" everything?